openwisp_utils/admin_theme/templates/openwisp_utils/email_template.html

```diff
@@ -0,0 +1,114 @@
+{% load i18n %}
+{% load l10n %}
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  {% block styles %}
+  <style type="text/css">
+    body {
+      min-width: 250px;
+      background: #ffe5e5;
+      margin: 0;
+      padding: 0;
+    }
+
+    .container{
+      background: #ffe5e5;
+      height: 100%;
+      min-height: 500px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .box{
+      width: auto;
+      padding: 60px 40px;
+      background-color: #fcfcfc;
+      box-sizing: border-box;
+      box-shadow: 0 15px 25px rgba(256,256, 256,.5);
+      border-radius: 10px;
+      margin: auto;
+    }
+
+    .logo-container{
+      text-align: center;
+      margin-bottom: 25px;
+    }
+
+    .logo {
+      max-width: 230px;
+    }
+
+    .call-to-action{
+      text-align: center;
+      margin-top: 30px;
+    }
+
+    .subject{
+      font-size: medium;
+      font-weight: 700;
+      margin: 20px 0;
+    }
+
+    .msg{
+      color: #1b1818;
+      margin: 20px 0;
+    }
+
+    .btn{
+      appearance: button;
+      text-decoration: none;
+      color: #fff!important;
+      background: #333;
+      border-radius: 4px;
+      border: 0;
+      padding: 10px 15px;
+      font-weight: 500;
+      font-size: 14px;
+      text-transform: uppercase;
+    }
+
+    .btn:hover, .btn:focus {
+      background: #000;
+      outline: 0 none;
+      background: rgba(51, 51, 51, 0.7);
+    }
+
+    a:link:hover {
+      text-decoration: none;
+    }
+  </style>
+  {% endblock styles %}
+</head>
+<body>
+  {% block body %}
+  <div class="container">
+    <div class="box">
+      {% block mail_header %}
+      <div class="logo-container">
+        <img src="{{ logo_url }}" alt="Logo" id="logo" class="logo">
+      </div>
+      {% if subject %}
+      <div class="subject">
+        {{ subject }}
+      </div>
+      {% endif %}
+      {% endblock mail_header %}
+      {% block mail_body %}
+      <div class="msg">
+        {{ message | safe }}
+      </div>
+      {% if call_to_action_url %}
+      <div class="call-to-action">
+        <a href="{{ call_to_action_url }}" class="btn">
+          {{ call_to_action_text }} </a>
+      </div>
```

Comment:
Can we make it a different block because doing this will add a possibility to put the call_to_action button between messages?
e.g.:
![image](https://user-images.githubusercontent.com/56037184/141118699-79e618c6-b066-4001-a2c0-6f5b22ebf541.png)

cc: @devkapilbansal @nemesisdesign

Reply:
Done :+1: